chia/types/blockchain_format/program.py

```diff
@@ -280,4 +280,5 @@ def _run(self, max_cost: int, flags, *args) -> Tuple[int, Program]:
         return cost, Program.to(ret)
 
 
-NIL = Program.from_bytes(b"\x80")
+NIL_PROGRAM = Program.from_bytes(b"\x80")
+NIL_SERIALIZED_PROGRAM = SerializedProgram.from_program(NIL_PROGRAM)
```

Comment:
not that it matters much, but I would have expected these to be set in the opposite order. The `SerializedProgram` is just a wrapper around the `b"\x80"` buffer.